net.kvirc.KVIrc.yaml

```diff
@@ -0,0 +1,24 @@
+app-id: net.kvirc.KVIrc
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-appdata-file: net.kvirc.KVIrc5.appdata.xml
+rename-desktop-file: net.kvirc.KVIrc5.desktop
+rename-icon: kvirc
+command: kvirc
+
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --filesystem=xdg-download
+  - --persist=.config/KVIrc
```

Comment:
are you sure you need this? This looks like the default XDG path in non sandbox.

Reply:
The KVIrc in first run show this dialog.
![image](https://user-images.githubusercontent.com/198756/171712209-59c9f42b-5dcb-46ea-913e-61b244e1959d.png)

then needs Download folder or uses override to home filesystem

btw ... i removed the persist and not saved the configuration

Reply:
This should be patched to use `$XDG_CONFIG_HOME` instead.

Reply:
I was in doubt... I don't know how to use this variable

`- --persist=$XDG_CONFIG_HOME/KVIrc`

Reply:
This is not going to work, and not what I meant in the first place. I mean that the application code needs to be changed to respect the variable.

If you want to go with `--persist`, it should be `--persist=.config/KVIrc`.